packages/app-harness/renative.json

```diff
@@ -6,7 +6,12 @@
         "path": "./secrets/privateConfigs.enc"
     },
     "plugins": {
-        "react-native-permissions": "3.10.1"
+        "react-native-permissions": "3.10.1",
+        "react-native": {
+            "android": {
```

Comment:
i think you also need `"source": "rnv"` as well, for react-native

Reply:
@mihaiblaga89  nothing changes when added `"source": "rnv"`